graph_def_editor/reroute.py

```diff
@@ -134,7 +134,7 @@ def _reroute_ts(ts0, ts1, mode, can_modify=None, cannot_modify=None):
   """Reroute the end of the tensors in each pair (t0,t1) in ts0 x ts1.
 
   This function is the back-bone of the Graph-Editor. It is essentially a thin
-  wrapper on top of the tf.Operation._update_input.
+  wrapper on top of the `gde.Node.replace_input`.
```

Comment:
I believe this is the equivalent, but can you verify @frreiss ?

Reply:
Yes, that's basically equivalent.
Nit: Remove "the" from this line or add the word "method" after `gde.Node.replace_input`.

Reply:
sure, that would be correct